kolibri/core/content/models.py

```diff
@@ -1,34 +1,35 @@
 """
-This is one of the Kolibri core components, the abstract layer of all contents.
-To access it, please use the public APIs in api.py
+These models are used in the databases of content that get imported from Studio.
 
-The ONLY public object is ContentNode
 
-Developer note: If you modify the schema here, it has implications for the content
-import pipeline, including Kolibri Studio where the imported content databases
-are created.
+*DEVELOPER WARNING regarding updates to these models*
 
-Currently, Kolibri Studio has a modified copy of this models.py for generating backwards compatible
-content databases. Changes here should be propagated to that copy in order to allow for
-generation of databases with the changed schema.
-TODO: (rtibbles) achieve this by abstract base models that are instantiated in both applications.
```

Comment:
Did you create an issue in lieu of this TODO?

Reply:
https://github.com/learningequality/kolibri/issues/4702